### PR TITLE
Text under h3 font issue

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/antora_ui/src/css/doc.css [hunk 1]
```diff
@@ -591,7 +591,7 @@ kbd,
   height: 100%;
 }
 
-.doc .paragraph p {
+.noversion-paragraph {
```
Comment (aknguyen7) [inline src/main/content/antora_ui/src/css/doc.css:594]: the .doc .paragraph p { is being use every where in the docs, just want to make sure we are not regress the behavior

**Comment by aknguyen7** [comment]
in demo1 site, the noversion still look the same compare to openliberty.io

https://demo1-openlibertyio.mybluemix.net/docs/20.0.0.7/noversion.html?ref=/docs/latest/reference/javadoc/liberty-jakartaee9.1-javadoc.html

the overview page
OL.io

image


demo1

image


It also obvious in this page

https://demo1-openlibertyio.mybluemix.net/docs/latest/development-mode.html

vs

https://openliberty.io/docs/latest/development-mode.html

File: src/main/content/antora_ui/src/css/doc.css (hunk 1 above)
Comment (Sreejith-Websphere) [reply src/main/content/antora_ui/src/css/doc.css:594]: this commit https://github.com/OpenLiberty/openliberty.io/commit/907bcab85bb44f98b4a15f078a1580f630fe40e5

I had created  .doc .paragraph p css added css to .doc .paragraph p { which effects everywhere, that was causing this font issue https://openliberty.io/docs/latest/network-hardening.html#_mutual_authentication

now I am adding and removing class specifically in no-version page, so can see the difference its not effects everywhere, its applies only in no-version page
https://demo1-openlibertyio.mybluemix.net/docs/latest/network-hardening.html#_mutual_authentication

**Comment by Sreejith-Websphere** [comment]
please test it now

File: src/main/content/antora_ui/src/css/doc.css (hunk 1 above)
Comment (aknguyen7) [reply src/main/content/antora_ui/src/css/doc.css:594]: lgtm

**Review by aknguyen7 — APPROVED** [review]

**Pull request merged**